### PR TITLE
how to build a product - android app

### PR DESCRIPTION
**Pull request by CityVibes**
Android part for [guide](https://docs.particle.io/guide/how-to-build-a-product/mobile-app/) on how to build your own mobile app.

### CODE REVIEW

File: src/content/guide/how-to-build-a-product/mobile-app.md [hunk 1]
```diff
@@ -201,13 +203,138 @@ You can find the source code for the Cloud SDK under our GitHub account:
...
+Well Done! You've just created a mobile basic app that can:
+1. Set up a Particle device interactively
+2. List the devices on the user's account
+3. Call a function on a device and report to the user
+
+**Modifying existing app**
+
+If you prefer to have a point of reference you're more than welcome to modify an existing app. Ideas for modifiable apps include:
+
+- [Particle Tinker open-source app](https://github.com/spark/photon-tinker-android)
```
Comment (monkbroc) [inline src/content/guide/how-to-build-a-product/mobile-app.md]: Update to https://github.com/particle-iot/photon-tinker-android

**Review by monkbroc — APPROVED** [review]
Great! Thanks for writing this up!

**Comment by monkbroc** [comment]
If it's ready to merge I can push the button.

**Comment by CityVibes** [comment]
@monkbroc yep ready

**Pull request merged**